### ISSUE TITLE
Xkblayout: AssertionError

### ISSUE THREAD

**Issue by m45t3r** [enkore/i3pystatus#523]
Xkblayout module seems broken after #492:

```
2017-01-12 00:50:05,471 [ERROR   ][i3pystatus.xkblayout.Xkblayout 76] Exception in Thread-3 at qui 12 jan 2017 00:50:05 BRST, module Xkblayout
Traceback (most recent call last):
  File "/home/m45t3r/.dotfiles/i3/.config/i3pystatus/venv/src/i3pystatus/i3pystatus/core/threading.py", line 68, in __call__
    self.workload()
  File "/home/m45t3r/.dotfiles/i3/.config/i3pystatus/venv/src/i3pystatus/i3pystatus/core/modules.py", line 285, in __call__
    self.run()
  File "/home/m45t3r/.dotfiles/i3/.config/i3pystatus/venv/src/i3pystatus/i3pystatus/xkblayout.py", line 79, in run
    "symbol": self._xkb.group_symbol,
  File "/home/m45t3r/.dotfiles/i3/.config/i3pystatus/venv/lib/python3.6/site-packages/xkbgroup/core.py", line 338, in group_symbol
    s_mapping = {symdata.index: symdata.symbol for symdata in self._symboldata_list}
  File "/home/m45t3r/.dotfiles/i3/.config/i3pystatus/venv/lib/python3.6/site-packages/xkbgroup/core.py", line 428, in _symboldata_list
    return _parse_symbols(b_symbol_str.decode(), ["pc", "inet", "group"])
  File "/home/m45t3r/.dotfiles/i3/.config/i3pystatus/venv/lib/python3.6/site-packages/xkbgroup/core.py", line 486, in _parse_symbols
    assert len(indices) == len(set(indices))    # No doubles
AssertionError
```

Problm seems to be in https://github.com/enkore/i3pystatus/blob/master/i3pystatus/xkblayout.py#L79. In fact, trying in terminal:

```
Python 3.6.0 (default, Dec 24 2016, 08:03:08) 
[GCC 6.2.1 20160830] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from xkbgroup import XKeyboard
>>> xkb = XKeyboard()
>>> xkb.group_symbol
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/m45t3r/.dotfiles/i3/.config/i3pystatus/venv/lib/python3.6/site-packages/xkbgroup/core.py", line 338, in group_symbol
    s_mapping = {symdata.index: symdata.symbol for symdata in self._symboldata_list}
  File "/home/m45t3r/.dotfiles/i3/.config/i3pystatus/venv/lib/python3.6/site-packages/xkbgroup/core.py", line 428, in _symboldata_list
    return _parse_symbols(b_symbol_str.decode(), ["pc", "inet", "group"])
  File "/home/m45t3r/.dotfiles/i3/.config/i3pystatus/venv/lib/python3.6/site-packages/xkbgroup/core.py", line 486, in _parse_symbols
    assert len(indices) == len(set(indices))    # No doubles
AssertionError
```

Maybe this is specific related to Python 3.6.0? Better report upstream (https://github.com/hcpl/xkbgroup)?

**Comment by enkore** [comment]
Looks like an upstream issue to me

**Comment by m45t3r** [comment]
Reported upstream: https://github.com/hcpl/xkbgroup/issues/1

**Comment by m45t3r** [comment]
Ok, the issue was really upstream.

However there is another issue. After calling the following:

```
setxkbmap us && pkill -SIGUSR1 -f i3pystatus
```
I need to restart i3pystatus since the indicator doesn't work (it doesn't update the layout and it doesn't change the layout on click either).
